src/stdune/sexp.ml

```diff
@@ -2,41 +2,10 @@ module Array = ArrayLabels
 module List = ListLabels
 module String = StringLabels
 
-type t = Sexp0.t =
+type t =
   | Atom of string
   | List of t list
 
-(* XXX otherwise the dependency isn't recorded by bootstrap *)
-module Sexp_intf = Sexp_intf
```

Comment:
@diml there's no need for these hacks anymore since i changed bootstrap to account for mli deps

Reply:
ack, thanks